### PR TITLE
Fix Collision bug

### PR DESCRIPTION
**Pull request by Rollczi**

### CODE REVIEW

**Comment by CDFN** [comment]
In my opinion determinating whether collisions should be enabled should be in configuration. Other option is getting it from bukkit's config.

File: src/main/java/net/dzikoysk/funnyguilds/element/IndividualPrefix.java [hunk 1]
```diff
@@ -219,6 +226,20 @@ public void initialize() {
         }
     }
 
+    private void registerSoloTeam(User member) {
```
Comment (dzikoysk) [inline src/main/java/net/dzikoysk/funnyguilds/element/IndividualPrefix.java:229]: `member`-> `soloUser`

File: src/main/java/net/dzikoysk/funnyguilds/element/IndividualPrefix.java [hunk 2]
```diff
@@ -125,6 +127,10 @@ protected void removeGuild(Guild guild) {
         if (team != null) {
             team.unregister();
         }
+
+        for (User u : guild.getMembers()) {
```
Comment (dzikoysk) [inline src/main/java/net/dzikoysk/funnyguilds/element/IndividualPrefix.java:131]: `u` -> `user`

**Review by dzikoysk — CHANGES_REQUESTED** [review]

**Comment by Rollczi** [comment]
```
[18:44:16 ERROR]: Could not pass event PlayerJoinEvent to SMC-Gildie v4.8.0 Tribute-54d99a3
org.bukkit.event.EventException: null
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:72) ~[server.jar:git-Paper-1618]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[server.jar:git-Paper-1618]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[server.jar:git-Paper-1618]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[server.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.PlayerList.onPlayerJoin(PlayerList.java:370) ~[server.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.PlayerList.a(PlayerList.java:183) ~[server.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.LoginListener.b(LoginListener.java:178) ~[server.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.LoginListener.e(LoginListener.java:68) ~[server.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.NetworkManager.a(NetworkManager.java:273) ~[server.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.ServerConnection.c(ServerConnection.java:150) ~[server.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:948) ~[server.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:423) ~[server.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:774) ~[server.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:666) ~[server.jar:git-Paper-1618]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_275]
Caused by: java.lang.IllegalArgumentException: Team name 'dziki_wonsz13_one' is longer than the limit of 16 characters
        at org.apache.commons.lang.Validate.isTrue(Validate.java:136) ~[server.jar:git-Paper-1618]
        at org.bukkit.craftbukkit.v1_12_R1.scoreboard.CraftScoreboard.registerNewTeam(CraftScoreboard.java:139) ~[server.jar:git-Paper-1618]
        at net.dzikoysk.funnyguilds.element.IndividualPrefix.registerOneTeam(IndividualPrefix.java:233) ~[?:?]
        at net.dzikoysk.funnyguilds.element.IndividualPrefix.initialize(IndividualPrefix.java:201) ~[?:?]
        at net.dzikoysk.funnyguilds.listener.PlayerJoin.onJoin(PlayerJoin.java:65) ~[?:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor533.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:70) ~[server.jar:git-Paper-1618]
        ... 14 more

```

**Comment by CDFN** [comment]
You have to check whether (nickname of player + suffix) is longer than 16 character. There's no workaround for it. I think you should consider choosing other way for creating team name (some identifier?).
Tip: Player's nickname max length is 16 characters.

**Review by dzikoysk — APPROVED** [review]

**Pull request merged**